packages/xdl/src/Project.js

```diff
@@ -184,6 +184,13 @@ async function _resolveGoogleServicesFile(projectRoot, manifest) {
     );
     manifest.android.googleServicesFile = contents;
   }
+  if (manifest.ios && manifest.ios.googleServicesFile) {
+    const contents = await fs.readFile(
+      path.resolve(projectRoot, manifest.ios.googleServicesFile),
+      'base64'
+    );
+    manifest.ios.googleServicesFile = contents;
```

Comment:
This code looks duplicated with the `expo-cli` one.

Reply:
Hmmm, I'm against creating utility functions only for the sake of not duplicating 2 lines of code :)

Reply:
It was a joke, not the funniest one, I agree